src/backend/app.js

```diff
@@ -21,10 +23,82 @@ app.use(cors());
 
 router.use("/meals", mealsRouter);
 
+// app.get("/my-route", (req, res) => {
+//   res.send("Hi friend");
+// });
+
+// /future-meals	Respond with all meals in the future (relative to the when datetime)
+
+app.get("/future-meals", async (req, res) => {
+  try {
+    const rows = await knex.raw("SELECT * FROM `meal` WHERE `when` >= NOW()");
+    rows[0].length === 0
```

Comment:
Query works fine but for my understanding - 
when row lenth is zero then you are returning 200 status code?
Isn't 404 more suitable when no records are found?